config/secrets.yml

```diff
@@ -0,0 +1,32 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+# Shared secrets are available across all environments.
+
+# shared:
+#   api_key: a1B2c3D4e5F6
+
+# Environmental secrets are only available for that specific environment.
+
+development:
+  secret_key_base: 61af8eee514ab07b95fd4155c03621fbec68fc7c48f2487fd897e4a39eee6fa659357795a047f999e826d82cb00b9ed7bccc3d63a6c0c51d1fa8fb04ce47d7b4
+
+test:
+  secret_key_base: 6fbcefb562f60bae4c91722cfa9b9c260c953458e25edc319c483d5a9aa448742d605e8baf82eac4cc9c17abd1a5c600b921d2af624eb4b33d8705a589ac4963
+
+# Do not keep production secrets in the unencrypted secrets file.
+# Instead, either read values from the environment.
+# Or, use `bin/rails secrets:setup` to configure encrypted secrets
+# and move the `production:` environment over there.
+
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
just to double check, have you tested signing in/up and resetting password simulating a real user after the rails upgrade?

Reply:
yeah, works fine. Already added this env value on production